src/components/Datatable/ItemsTable.svelte

```diff
@@ -80,6 +80,8 @@ $: selectedItemNames = checkedItems.map((item) => item.name)
 $: sortedItemsArray = currentColumn.numeric
   ? sortByNum(currentColumn.path, items, ascending)
   : sortByString(currentColumn.path, items, ascending)
+$: allCheckedItemsAreDraft =
+  checkedItems.length > 0 && checkedItems.every((item) => item.coverage_status === ItemCoverageStatus.Draft)
```

Comment:
Not sure if the length check is needed, it might be better to check if any items are not Draft instead. That way, if all items in table are in draft status, the button will show as "delete", otherwise it will be shown as "end coverage". Something like:

```suggestion
  items.some((item) => item.coverage_status !== ItemCoverageStatus.Draft) &&
  checkedItems.every((item) => item.coverage_status === ItemCoverageStatus.Draft)
```

Reply:
The code works but still shows "End Coverage" when I only have a Draft Item. Funny that running [].every(callback) or [].some(callback) returns true, I tested it out in a sandbox. think I'll leave it as is since It will probably require yet another condition to have the benefit of showing "Delete" before selecting the lone Draft item.